core-android/src/main/java/com/uber/sdk/android/core/auth/LoginManager.java

```diff
@@ -185,11 +186,21 @@ public void loginForAuthorizationCode(@NonNull Activity activity) {
     }
 
     /**
-     * @return {@link AccessTokenManager} that is used.
+     * @return {@link AccessTokenStorage} that is used.
+     * @deprecated Use {@link LoginManager#getAccessTokenStorage()}
      */
+    @Deprecated
     @NonNull
-    public AccessTokenManager getAccessTokenManager() {
-        return accessTokenManager;
+    public AccessTokenStorage getAccessTokenManager() {
+        return accessTokenStorage;
```

Comment:
How do nullability annotations work on an implicit cast, here? 

Seems like this could cause a crash. Do you want to assert, since you can't really change the API? Otherwise a developer could ignore the @deprecated warning and then get NPE in production. 

The scenario I'm worried about is where a developer takes advantage of the interface and implements their own storage, and forgets to update the call to `getAccessTokenManager`. I think it's a remote possibility, since they would get a crash anyways since it should be a reasonably common code path.